hoot-core-test/src/test/cpp/hoot/core/io/PbfReaderTest.cpp

```diff
@@ -554,6 +555,47 @@ class PbfReaderTest : public CppUnit::TestFixture
 
     CPPUNIT_ASSERT_EQUAL(40, numberOfElements);
   }
+
+  void runReadSortTypeTest()
```

Comment:
Is there a test for when it the sorted flag isn't set and values are out of order?